content/installation/node/YAML-properties-node.md

```diff
@@ -0,0 +1,162 @@
+<!--
+title: "YAML configuration properties for the Node agent "
+description: "Instructions for configuring Node agent properties via YAML file"
+tags: "installation node agent YAML configuration rules properties"
+-->
+
+Contrast support YAML-based configuration for the Node agent. This allows you to store configuration on disk with environment variable and CLI args. overrides.
+
+> **Note:** While property formatting in YAML configuration files is shared by all Contrast agents, each agent must use its specified file. 
+
+## Load Path 
+
+Configuration values use the following order of precedence: 
+
+1. Corporate rule (e.g., expired license overrides `assess.enable`)
+2. Command line value
+3. Specific environmental variable
+4. Generic environment variable value
+5. User configuration file value
+6. Contrast UI value
+7. Default value 
+
+Go to the [Node YAML Template](installation-nodeconfig.html#node-template) for fully formatted properties that you can copy and use in your own configuration files. 
+
+## Set as System Properties
+
+You can also set all of the following YAML properties as system properties. Derive the system property key from the YAML by joining every node with a "." until you reach the bottom property. 
+
+> **Example:** If you want to override the `contrast` property, as given below, you can set `-Dcontrast.enable=false` as a system property.
+  * contrast: 
+    * enable: `true`
+
+<!-- The precedence and system properties example are copied from Java. Edit or delete as needed.  -->
+
```

Comment:
@meandthemoon Can you confirm that this is correct?

Reply:
Yep - the system properties bit was missed. I'll delete it.